loader/generated/vk_object_types.h

```diff
@@ -186,6 +190,8 @@ const VkDebugReportObjectTypeEXT get_debug_report_enum[] = {
     VK_DEBUG_REPORT_OBJECT_TYPE_UNKNOWN_EXT,   // kVulkanObjectTypePerformanceConfigurationINTEL
     VK_DEBUG_REPORT_OBJECT_TYPE_UNKNOWN_EXT,   // kVulkanObjectTypeIndirectCommandsLayoutNV
     VK_DEBUG_REPORT_OBJECT_TYPE_BUFFER_COLLECTION_FUCHSIA_EXT,   // kVulkanObjectTypeBufferCollectionFUCHSIA
+    VK_DEBUG_REPORT_OBJECT_TYPE_UNKNOWN_EXT,   // kVulkanObjectTypeMicromapEXT
+    VK_DEBUG_REPORT_OBJECT_TYPE_UNKNOWN_EXT,   // kVulkanObjectTypeOpticalFlowSessionNV
```

Comment:
This looks suspicious...

Reply:
Since debug report is deprecated in favor of debug utils I think they stopped adding new object enums for it. Let me find a better reference for that information

Reply:
That makes sense, and I see that other entries are also missing enums.

Reply:
I couldn't find any concrete policy that we stopped updating `VkDebugReportObjectTypeEXT`, except that it is in fact officially deprecated. The Vulkan style guide explicitly mentions adding `VkObjectType` values for new handle types but makes no mention of `VkDebugReportObjectTypeEXT` https://registry.khronos.org/vulkan/specs/1.3/styleguide.html#_extension_handles_objects_enums_and_typedefs